### ISSUE TITLE
Tutorial reduction_sequence_std_stars

### ISSUE THREAD

**Issue by miguelglezb** [pykoala/koala#414]
First cell after 
# Corrections
fails because `pykoala.mplstyle` is not a valid package style.

**Comment by miguelglezb** [comment]
To avoid making a new issue, this problem is also present in tutorial `create_combined_throughput`

**Comment by PabloCorcho** [comment]
This is a problem that @angelrls encountered in the past but I never got to fix. There seems to be a problem with a latex dependency within matplotlib. My guess is that lines 63 and 73 of `src/plotting/pykoala.mplstyle` are the ones responsible for this issue. Can you please have a look if by removing those the problem is solved?

**Comment by miguelglezb** [comment]
I fixed the problem by copying `pykoala.mplstyle` in `~/.local/lib/python3.10/site-packages/pykoala/plotting/`. However, this worked because I installed pykoala in my machine with 
``` pip install . ```
The problem is that the notebooks don't work unless I install pykoala locally due to issues when importing pykoala modules from relative paths. 

There is two ways to fix this:
- Installing locally and modifying the paths of `pykoala.mplstyle` (also of `telluric_lines.txt` that present the same error in the notebook) for consistency.
- Not installing locally, but adding (where needed):
 ``` 
import sys
import os

sys.path.append('../')
```

There are pros and cons with each option, but I leave @PabloCorcho to decide which, if any, of these options is the most appropiate for this.

**Comment by PabloCorcho** [comment]
I duplicate this comment from #415 :

I have created a [branch](https://github.com/pykoala/koala/tree/414_fix_mplstyle) that includes the mplstyle in the MANIFEST.in file, so it is also included during the installation. Hopefully this solves this issue. @miguelglezb could you please try it in your environment?

**Comment by miguelglezb** [comment]
Yes, that fixes it. However the tutorial still fails because it can't find `input_data/sky_lines/telluric_lines.txt`. I suspect this is caused by issues with `pykoala/corrections/sky.py` in line 632 (relative paths?)

**Comment by miguelglezb** [comment]
In last cell of the notebook, there is no key `'response'`, maybe use `results['HILT600']['response_fig']`

**Comment by miguelglezb** [comment]
The cell works with `results['HILT600']['response'], results['HILT600']['wavelength'])`

**Comment by miguelglezb** [comment]
I think we can close this issue, as all cells work now. I won't apply the changes until I finish with the documentation PR, @PabloCorcho @paranoya

**Comment by PabloCorcho** [comment]
I prefer that we close this issue after the PR is merged.
